interfaces/builtin/wireless-control.go

```diff
@@ -0,0 +1,136 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+
+const  wirelessControlConnectedPlugAppArmor = `
+# Description: Can configure wireless networking. This is restricted because it gives
+# wide, privileged access to wireless networking and should only be used with trusted
+# apps.
+# Usage: reserved
+
+capability net_admin,
+capability setgid,
+
+network netlink dgram,
+network bridge,
+network packet,
+
+@{PROC}/@{pid}/net/ r,
+@{PROC}/@{pid}/net/** r,
+
+# used by sysctl, et al
+@{PROC}/sys/ r,
+@{PROC}/sys/net/ r,
+@{PROC}/sys/net/dev/ r,
+@{PROC}/sys/net/core/ r,
+@{PROC}/sys/net/core/** rw,
+@{PROC}/sys/net/ipv{4,6}/ r,
+@{PROC}/sys/net/ipv{4,6}/** rw,
+@{PROC}/sys/net/netfilter/ r,
+@{PROC}/sys/net/netfilter/** rw,
+@{PROC}/sys/net/nf_conntrack_max rw,
+
+# wireless tools
+/{,usr/}{,s}bin/iw ixr,
+/{,usr/}{,s}bin/iwconfig ixr,
+/{,usr/}{,s}bin/iwevent ixr,
+/{,usr/}{,s}bin/iwgetid ixr,
+/{,usr/}{,s}bin/iiwlist ixr,
+/{,usr/}{,s}bin/iwpriv ixr,
+/{,usr/}{,s}bin/iwspy ixr,
+
+/{,usr/}{,s}bin/dnsmasq ixr,
+/{,usr/}{,s}bin/hostapd ixr,
+
+/dev/rfkill r,
+/sys/class/net/ r,
+/sys/class/net/** r,
+/bin/sync ixr,
+
+#include <abstractions/nameservice>
+
+# DBus accesses
+#include <abstractions/dbus-strict>
+
+# Allow access to wpa-supplicant for managing WiFi networks
+dbus (receive, send)
```

Comment:
For which reason do we have dbus rules in here? I don't see any reasons for this interface caring them. Its just about enabling raw wireless management and nothing else.